### DISCUSSION TITLE
Multiple tables on same page

### DISCUSSION THREAD

**Discussion by rudolfbruder** [rappasoft/laravel-livewire-tables#342] [Q&A]
Hi,

Is there a way how to use multiple tables on same page and apply filter as well?

I know that i have to set pageName and tableName for pagination to work but that does not work for filters in case they have different columns. Is there some solution to this please?

This is my url http://xxxxx/settings/eshop/2?widgets=2&sorts[location]=asc but i also have another table called banners in there which does not have location column and the page crashes on reload.

Thank you!

**Comment by rappasoft** [comment]
Right now this is a known limitation. Multiple tables only work in the sense of pagination.

**Reply by rudolfbruder** [reply]
Thank you for quick response

**Comment by lrljoe** [comment]
This works in v2+
